Saving data in db

lucianoeli opened the pull request.

lucianoeli committed:
Saving data in db

capraber commented on data/src/main/java/com/globant/data/repositories/MarvelCharacterRepositoryImpl.kt:
```diff
@@ -1,36 +1,39 @@
+    }
+
+    private fun insertOrUpdateCharacter(character: MarvelCharacter) {
+        val realm = Realm.getDefaultInstance()
```
Why you use a val if you don't do anything with it?

capraber commented on data/src/main/java/com/globant/data/service/CharacterService.kt:
```diff
@@ -0,0 +1,27 @@
+
+        fun getCharacterById(id: Int): Result<MarvelCharacter> {
+            val callResponse = api.createService(MarvelApi::class.java).getCharacterById(id)
+            val response = callResponse.execute()
```
add NPE to call Response

SartiMau replied:
fixed in next PR

SartiMau replied:
done

SartiMau committed:
Comment fixes

Tested in: Samsung Galaxy S8
Signed off by: Mauro Sarti mauro.sarti@globant.com
